dashboard/legacy/test/middleware/files_api/test_public_thumbnails.rb

```diff
@@ -10,175 +10,23 @@ def setup
     Aws::S3::Client.expects(:new).never
   end
 
-  def test_adult_thumbnail
-    ImageModeration.stubs(:rate_image).once.returns :adult
-
-    with_project_type('applab') do |channel_id|
-      get "/v3/files-public/#{channel_id}/#{@thumbnail_filename}"
-
-      # Responds with a 404, like we do for flagged content
-      assert not_found?
-
-      # Includes content rating metadata in the response that the client can read
-      assert_equal 'adult', last_response['x-cdo-content-rating']
-
-      # Response is cached for an hour
-      assert_equal 'public, max-age=3600, s-maxage=1800', last_response['Cache-Control']
-
-      # Flags the project as abusive.
-      value = Projects.get_abuse(channel_id)
-      assert_equal 15, value
-
-      # Flags the thumbnail as abusive
-      thumbnail = FileBucket.new.get(channel_id, @thumbnail_filename)
-      metadata = thumbnail[:metadata]
-      thumbnail_abuse = [metadata['abuse_score'].to_i, metadata['abuse-score'].to_i].max
-      assert_equal 15, thumbnail_abuse
-    end
-  end
-
-  def test_racy_thumbnail
-    ImageModeration.stubs(:rate_image).once.returns :racy
-
-    with_project_type('applab') do |channel_id|
-      get "/v3/files-public/#{channel_id}/#{@thumbnail_filename}"
-
-      # Responds with a 404, like we do for flagged content
-      assert not_found?
-
-      # Includes content rating metadata in the response that the client can read
-      assert_equal 'racy', last_response['x-cdo-content-rating']
-
-      # Response is cached for an hour
-      assert_equal 'public, max-age=3600, s-maxage=1800', last_response['Cache-Control']
-
-      # Flags the project as abusive.
-      value = Projects.get_abuse(channel_id)
-      assert_equal 15, value
-
-      # Flags the thumbnail as abusive
-      thumbnail = FileBucket.new.get(channel_id, @thumbnail_filename)
-      metadata = thumbnail[:metadata]
-      thumbnail_abuse = [metadata['abuse_score'].to_i, metadata['abuse-score'].to_i].max
-      assert_equal 15, thumbnail_abuse
-    end
-  end
-
-  def test_everyone_thumbnail
-    ImageModeration.stubs(:rate_image).once.returns(:everyone).with do |file, _, _|
-      # Real ImageModeration reads the IO stream in order to do its work.
-      # As a result, the caller needs to rewind the stream before using it as
-      # the response body.
-      # Here, I'm ensuring our stubbed ImageModeration simulates that behavior.
-      file.read
-      # Return true so this expectation matches any arguments.
-      true
-    end
-
+  def test_cached_thumbnail
     with_project_type('applab') do |channel_id|
       get "/v3/files-public/#{channel_id}/#{@thumbnail_filename}"
       assert successful?
-      assert_equal 'public, max-age=3600, s-maxage=1800', last_response['Cache-Control']
+      cache_control = last_response['Cache-Control']
```

Comment:
Did this change somehow?

Reply:
The response I got locally also included '`must-revalidate`'. This test is failing on Drone although passing locally and unsure why the `Cache-Control` tag is different on drone. Will post in #dev-support. Lmk if you have any ideas!